### ISSUE TITLE
add images to diaries

### ISSUE THREAD

**Issue by whoeveritis** [danielroehrig/diary#60]
hallo, are there plans to add images to diary entries, as we have in nextcloud text? Links are great, but a visible photo in preview would be even better. (Nextcloud text gets very slow if it contais say 25 2MB images) Plus it would be neat to self organize the images (Nextcloud text uses a default invisible folder here)

**Comment by danielroehrig** [comment]
Do you mean notes? The whole notes app is awesome and I want to ~~steal~~ be inspired by their feature set. Unfortunately, I currently have very little time and am a lousy, bad, no good frontend dev. I am still hoping that one day someone will offer to do the frontend for me. But yes. I want to have images in my diary as well.

**Comment by whoeveritis** [comment]
yes, well it used to be #notes I think and now it's #nextcloud_text and it writes .md files. I found #typora app which does diary tasks nicely over #nextcloud_sync. My problem is Nextcloud so laggy in browser apps.

**Comment by graphius** [comment]
If you want a more "full featured" app, you could try [notes](https://apps.nextcloud.com/apps/notes)

I like the simplicity of the diary app the way it is.

**Comment by SteveDinn** [comment]
I kind of rolled my own solution to this by creating a `.Diary` folder off the root of my file store and linking to the attachments I store there.

![image](https://github.com/danielroehrig/diary/assets/3911507/b403b12b-a8a8-4d5f-96c0-6f7a75789a7d)
